<launch>
  <node pkg="gto" type="ground_texture_odometry_node" name="ground_texture_odometry" output="screen">
  <remap from="/image_raw" to="/raspicam_node/image/compressed"/>
  <remap from="/camera_info" to="/raspicam_node/camera_info"/>
  </node>
</launch>
